docs/reference/resources.rst

```diff
@@ -43,6 +43,7 @@ Methods
    :toctree: generated
 
    client.Client.from_config
+   client.Client.solvers
```

Comment:
I already see this https://docs.ocean.dwavesys.com/projects/cloud-client/en/latest/reference/resources.html
![image](https://user-images.githubusercontent.com/34041130/46834869-739d2380-cd61-11e8-96da-d917b7d4022e.png)

Reply:
Yes, because I already merged this PR. :)